resources/views/ItemList/Components/ItemSearch.twig

```diff
@@ -1,5 +1,5 @@
 <script type="x/template" id="vue-findologic-item-search">
-    <form method="GET" action="/search">
+    <form method="GET" action="/search" onsubmit="return onFlFormSubmit(event)">
```

Comment:
Isn't it possible to make a simple `onsubmit="return false;"` here?

Reply:
true, that works as well. Less overhead and much cleaner. Thanks for your suggestion.